seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-jdbc/src/main/java/org/apache/seatunnel/flink/jdbc/sink/JdbcSink.java

```diff
@@ -151,4 +178,44 @@ public void outputBatch(FlinkEnvironment env, DataSet<Row> dataSet) {
                 .finish();
         dataSet.output(format);
     }
+
+    @Override
+    public void close() throws Exception {
+        executePostSql();
+    }
+
+    private void executePreSql() {
+        if (StringUtils.isNotBlank(preSql)) {
+            LOGGER.info("Starting to execute pre sql: \n {}", preSql);
+            try {
+                executeSql(preSql);
+            } catch (SQLException e) {
+                LOGGER.error(String.format("Execute pre sql failed, pre sql is : \n %s \n", preSql), e);
```

Comment:
Please pay attention to your log format, we just only use log.error(".....{}",preSql,e). Because printf-style format strings are interpreted at runtime, rather than validated by the compiler, they can contain errors that result in the wrong strings being created.

Reply:
if use log.error(".....{}",preSql,e), the exception can not been print well. 
<img width="449" alt="image" src="https://user-images.githubusercontent.com/19817318/167056759-2bedea1f-95f8-4f04-987c-3966175e7cc4.png">
There is no matching method to handle exceptions well

Reply:
The last arg will be parsed as throwable.